### PR TITLE
[ICON] [SWE-1850] [SWE-1861] [SWE-1764] Create IconGuideScreen, fix icons, fix constants

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**
Screenshot 2022-08-03 at 14 01 30

### CODE REVIEW

File: src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { AppIconLogo } from '~/app/domains'
+import { AppIconLogo } from '..'
```
Comment (Olexandr25) [inline src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.js:1]: Please, write full path

File: src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
-import { ContentScreen } from '~/app/components'
-import IconList from '~/app/domains/Iconography/components/IconList'
+import { ContentScreen, PositionedContainer } from 'components'
```
Comment (Anhelina4) [inline src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js:1]: rewrote with ~/app/

File: src/app/domains/Iconography/components/IconList/IconList.js [hunk 3]
```diff
@@ -1,15 +1,29 @@
-import { ICONS_LIST } from 'constants'
+import './IconList.vars.module.css'
+
 import IconListStyled from './IconList.styled'
-import { IconSimpleView } from 'domains/Iconography/components'
+import IconSimpleView from '../IconSimpleView'
```
Comment (Olexandr25) [inline src/app/domains/Iconography/components/IconList/IconList.js:4]: Please, write full path

File: src/app/domains/Imagery/components/ImageryView/ImageryView.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { IMG_DIRECTION, INSTA_POST, MOODBOARD } from '~/app/__constants__'
+import { IMG_DIRECTION, INSTA_POST, MOODBOARD } from '../../__constants__'
```
Comment (Olexandr25) [inline src/app/domains/Imagery/components/ImageryView/ImageryView.js:1]: Please, write like previous

File: src/app/domains/Typography/components/Alphabet/Alphabet.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import './Alphabet.vars.module.css'
 
-import { ALPHABET } from '~/app/__constants__'
+import { ALPHABET } from '../../__constants__'
```
Comment (Olexandr25) [inline src/app/domains/Typography/components/Alphabet/Alphabet.js:3]: Please, write like previous

File: src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js (hunk 2 above)
Comment (Olexandr25) [inline src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js:1]: Write path, '~/app/components'

File: src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js [hunk 6]
```diff
@@ -1,5 +1,7 @@
...
+
+import { ICONS_LIST } from '~/app/domains/Iconography/__constants__'
+import IconList from 'domains/Iconography/components/IconList'
```
Comment (Olexandr25) [inline src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamplesScreen.js:4]: ~app/domains

File: src/app/screens/Iconography/IconographyContent/IconographyGuide/IconographyGuideScreen.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import { ContentScreen, PositionedContainer } from '~/app/components'
+import {
+  IconAdvancedView,
+  IconList
+} from '~/app/domains/Iconography/components'
+
+import { ICONS_GUIDE_LIST } from '~/app/domains/Iconography/__constants__'
```
Comment (Olexandr25) [inline src/app/screens/Iconography/IconographyContent/IconographyGuide/IconographyGuideScreen.js:7]: ~app/domains

**Review by whoiam460 — APPROVED** [review]

**Review by Olexandr25 — APPROVED** [review]

**Pull request merged**